src/lib/contract.ts

```diff
@@ -49,14 +50,24 @@ export class Contract {
     return readJSON(path.resolve(this.artifactsPath, `${this.moduleName}.json`));
   }
 
+  async getBuildMode(): Promise<string> {
```

Comment:
Instead of a string promise we can return a BuildMode enum promise like [here](https://github.com/paritytech/cargo-contract/blob/37f12c82428eca708891f20fa369757796ff082b/crates/build/src/args.rs#L193-L201) in cargo-contract:

> ```rust
> /// The mode to build the contract in.
> #[derive(
>     Eq, PartialEq, Copy, Clone, Debug, Default, serde::Serialize, serde::Deserialize,
> )]
> pub enum BuildMode {
>     /// Functionality to output debug messages is build into the contract.
>     #[default]
>     Debug,
>     /// The contract is built without any debugging functionality.
>     Release,
>     /// the contract is built in release mode and in a deterministic environment.
>     Verifiable,
> }
> ```